zendesk/webhook.go

```diff
@@ -28,10 +28,18 @@ type Webhook struct {
 	UpdatedBy      string                 `json:"updated_by,omitempty"`
 }
 
+type WebhookCredentials struct {
+	HeaderName  string `tfsdk:"name,omitempty"`
+	HeaderValue string `tfsdk:"value,omitempty"`
+	Username    string `tfsdk:"username,omitempty"`
+	Password    string `tfsdk:"password,omitempty"`
+	Token       string `tfsdk:"token,omitempty"`
+}
+
 type WebhookAuthentication struct {
-	Type        string      `json:"type"`
-	Data        interface{} `json:"data"`
-	AddPosition string      `json:"add_position"`
+	Type        string              `json:"type"`
+	Data        *WebhookCredentials `json:"data"`
```

Comment:
Why are we using a pointer here?

Reply:
I just saw that's what the author was doing for the other objects, so I copied.  🙂

Reply:
Oh yeah I see that now.